services/GoogleAdSense.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Google AdSense",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.google.com/adsense/tc/2018/UnitedKingdom.html",
+      "select": "body"
+    },
+    "Program Policies": {
+      "fetch": "https://support.google.com/adsense/answer/48182",
+      "select": ".article-container",
+      "remove": ".action-button, .zippy"
+    },
+    "Parent Organization Privacy Policy": {
+      "fetch": "https://policies.google.com/privacy?hl=en-US",
+      "filter": [
+        "removeUTMfromUrls"
+      ],
+      "select": "div[role=article]"
+    },
+    "Brand Guidelines": {
+      "fetch": "https://support.google.com/adsense/answer/6370057",
```

Comment:
As internal links does not work for use, maybe remove the table of content

Reply:
Solved !